applications/smart-distancing/config-jetson.ini

```diff
@@ -13,9 +13,11 @@ ImageSize: 300,300,3
 ModelPath: 
 ClassID: 0
 MinScore: 0.25
-DistThreshold: 0.1
 
 [PostProcessor]
 MaxTrackFrame: 5
 NMSThreshold: 0.98
-
+; distance threshold for smart distancing in (cm)
+DistThreshold: 150
+; ditance mesurement method, 1: compare center of pedestrian boxes together, 2: compare four corresponding points of pedestrian boxes and get the minimum of them.
+DistMethod: 1 
```

Comment:
It is better to use informative names to distinguish different distance methods

Reply:
There is an informative description above the names, isn't it enough? what is your suggestion about the names?

Reply:
Description is great, however I suggest to use names instead of numbers.
I suggest `CenterPointsDistance` for number 1 and `FourCornerPointsDistance` for second one.

Reply:
modified, thanks